packages/storybook/src/documentation/pages/sorting.stories.mdx

```diff
@@ -114,18 +116,64 @@ const columns: Columns = [
 
 <SortIndexDemo />
 
+### Using the `sortModel` options
+
+XGrid also offers to override the sortModel using its `sortModel` property of its `options` React prop.
+**Note** The `sortModel` property is an array and its order will be used as the `sortIndex` property described above.
+
+```tsx
+<XGrid
+  rows={rows}
+  columns={columns}
+  options={{
+    sortModel = [
+      { field: 'age', sortDirection: 'desc' },
+      { field: 'name', sortDirection: 'asc' },
+    ],
+  }}
+/>
+```
+
+<SortModelOptionDemo />
+
+## Server-side sorting
+
+XGrid supports both client and server-side pagination. By default, sorting works on the client-side. To switch it to server-side,
+set the property `sortingMode` to `server` as string or you can use our `FeatureModeConstant`.
```

Comment:
IMHO, we don't need to export not document `FeatureModeConstant`  it feels like using a string will be simpler, but no strong preference.